app/src/ui/discard-changes/discard-selection-dialog.tsx

```diff
@@ -80,12 +79,13 @@ export class DiscardSelection extends React.Component<
         type="warning"
       >
         <DialogContent>
-          <p>
-            Are you sure you want to discard the selected changes to
-            <Monospaced>
+          <p>Are you sure you want to discard the selected changes to</p>
```

Comment:
Just added a colon to match the other discard changes dialog:

![image](https://user-images.githubusercontent.com/408035/86821454-b7ce6c80-c08a-11ea-9b25-d62fcfc0dd0a.png)


```suggestion
          <p>Are you sure you want to discard the selected changes to:</p>
```